packages/graphql-server-lambda/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "graphql-server-lambda",
+  "version": "0.4.3",
+  "description": "Production-ready Node.js GraphQL server for AWS Lambda",
+  "main": "dist/index.js",
+  "scripts": {
+    "compile": "tsc",
+    "prepublish": "npm run compile"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/apollostack/graphql-server/tree/master/packages/graphql-server-lambda"
+  },
+  "keywords": [
+    "GraphQL",
+    "Apollo",
+    "Server",
+    "Lambda",
+    "Javascript"
+  ],
+  "author": "Jonas Helfer <jonas@helfer.email>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/apollostack/graphql-server/issues"
+  },
+  "homepage": "https://github.com/apollostack/graphql-server#readme",
+  "dependencies": {
+    "graphql-server-core": "^0.4.3",
+    "graphql-server-module-graphiql": "^0.4.3"
+  },
+  "devDependencies": {
+    "@types/aws-lambda": "0.0.5",
```

Comment:
no dep (or peer-dep) for aws-lambda?
i'm not familiar with the node library so i'm not sure ;)